packages/sync-actions/src/order-actions.js

```diff
@@ -37,3 +38,56 @@ export function actionsMapDeliveries(diff, oldObj, newObj) {
 
   return handler(deliveriesDiff, oldObj.shippingInfo, newObj.shippingInfo)
 }
+
+export function actionsMapReturnsInfo(diff, oldObj, newObj) {
+  const returnInfoDiff = diff.returnInfo
+  if (!returnInfoDiff) return []
+
+  const handler = createBuildArrayActions('returnInfo', {
+    [CHANGE_ACTIONS]: (oldSReturnInfo, newReturnInfo) => {
+      const updateActions = Object.keys(returnInfoDiff).reduce(
+        (itemActions, key) => {
+          const { items = {} } = returnInfoDiff[key]
+          if (Object.keys(items).length > 0) {
+            return [
+              ...itemActions,
+              ...Object.keys(items).reduce((actions, index) => {
```

Comment:
wouldn't it be nicer if this was `Object.values` alternatively `Object.entries`? then a `items[index]...` wouldn't be needed

Reply:
I'm using the key to get the updated prop from the `newReturnInfo`, while I'm iterating over the `returnInfoDiff` item's key